pre-reg-113-115/src/main/java/com/reencryptutility/ReEncryptUtilityApplication.java

```diff
@@ -5,20 +5,27 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceTransactionManagerAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.FilterType;
 import org.springframework.web.client.RestTemplate;
 
 @SpringBootApplication
-@ComponentScan(basePackages={ "io.mosip.commons.khazana.*" ,"com.reencryptutility.*"}, excludeFilters = {
+@ComponentScan(basePackages={ "io.mosip.commons.khazana.*" ,"com.reencryptutility.*", "io.mosip.*", "io.mosip.kernel.auth.defaultadapter"},
```

Comment:
io.mosip.kernel.auth.defaultadapter scan using properties file

Reply:
Removed the io.mosip.kernel.auth.defaultadapter